src/js/App/Sidenav/Navigation.js

```diff
@@ -189,10 +189,18 @@ export const Navigation = () => {
     }
   };
 
+  const settingsWithSections = settings.reduce((acc, item) => {
+    const section = acc.find((i) => i.section === item.section) || { items: [], section: item.section };
```

Comment:
I would cimplify this
```suggestion
    const section = acc.find(({ section }) => section === item.section) || { items: [], section: item.section };
```